pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/PinotResultSet.java

```diff
@@ -193,13 +193,19 @@ public InputStream getAsciiStream(int columnIndex)
   public BigDecimal getBigDecimal(int columnIndex, int scale)
```

Comment:
i was wondering if we should just implement the non deprecated version of the API `getBigDecimal(int columnIndex/columnLabel)`?

Reply:
It's a good question.  We want to be as minimally invasive as possible, so if your team feels that to be a necessary change, then I think it's best if they create a new issue to address it.

Reply:
:+1: